src/platform/packages/shared/kbn-monaco/src/languages/esql/language.ts

```diff
@@ -79,12 +79,11 @@ export const ESQLLang: CustomLangModuleType<ESQLCallbacks> = {
       triggerCharacters: ESQL_AUTOCOMPLETE_TRIGGER_CHARS,
       async provideCompletionItems(
         model: monaco.editor.ITextModel,
-        position: monaco.Position,
-        context: monaco.languages.CompletionContext
```

Comment:
Nice seeing this go, I was always confused with this context object.

Reply:
Yes, thank you for doing this. Don't remember why I didn't remove in https://github.com/elastic/kibana/pull/223065 🙇 

I guess this means we could consider removing it from our test suite, too.